packages/quantic/force-app/main/default/lwc/quanticResultAction/quanticResultAction.js

```diff
@@ -65,11 +53,35 @@ export default class QuanticResultAction extends LightningElement {
   _selected;
   /** @type {boolean} */
   _loading;
+  
+  /**
+   * Specifies whether the result action button is in selected state or not.
+   * @api
+   * @type {boolean}
+   */
+  @api 
+  get selected() {
+    return this._selected;
+  }
 
-  connectedCallback() {
-    this.setSelected(this.selected);
-    this.setLoading(this.loading);
+  set selected(isSelected) {
+    this._selected = isSelected;
+  }
 
+  /**
+   * Specifies whether the result action button is in loading state or not.
+   * @api
+   * @type {boolean}
+   */
+  @api 
+  get loading() {
+    return this._loading;
+  }
+  set loading(isLoading) {
```

Comment:
minor nitpick but here we have the get and set with no new line in between and those for `selected` have a new line. either is fine but ideally not both. 😅